lib/cookieContentHandler.js

```diff
@@ -25,16 +25,17 @@
 
 var crypto = require('crypto');
 var createBuffer = require('./jwe').createBuffer;
+var sameSiteNotAllowed = require('./aadutils').sameSiteNotAllowed;
 
 /*
  * the handler for state/nonce/policy
  * @maxAmount          - the max amount of {state: x, nonce: x, policy: x} tuples you want to save in cookie
  * @maxAge            - when a tuple in session expires in seconds
- * @cookieEncryptionKeys  
+ * @cookieEncryptionKeys
  *                    - keys used to encrypt and decrypt cookie
  * @domain            - sets the cookie's domain
- */ 
-function CookieContentHandler(maxAmount, maxAge, cookieEncryptionKeys, domain) {
+ */
+function CookieContentHandler(maxAmount, maxAge, cookieEncryptionKeys, domain, cookieSameSite) {
```

Comment:
So is this off by default? I think it should be on, with option to disable.

Reply:
The concern was that it would need to be a major release if it was on by default. 
Not all http frameworks support the samesite option, so if i a customer was using express 3 for instance, this would throw.
They may be using other frameworks as well such as koa, hapi, restify along with passport.

Making this opt in means that when they go to update their app for samesite, this will be an option they need to confirm works with their dependency chain.

Reply:
Thanks, I appreciate the explanation.  We'll let developers know that this is a required action on their part in the notes.

Reply:
I will update readme as a part of the version cut to make sure this is all explained well